lib/scheduler.js

```diff
@@ -0,0 +1,55 @@
+const fs = require('fs');
+const path = require('path');
+const tempDir = require('temp-dir');
+
+const buildPath = projectDir => path.join(tempDir, 'ava', path.basename(projectDir));
```

Comment:
IIRC there is a "cache dir" concept in `api.js`. Perhaps this should use that same directory?